app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -833,7 +833,7 @@ def build_toolbar_hide_button(id)
         return true if @record.host && @record.host.vmm_product.downcase == "workstation"
       when "miq_template_refresh"
         return true if @record && !@record.ext_management_system && !(@record.host && @record.host.vmm_product.downcase == "workstation")
-      when "miq_template_scan"
+      when "miq_template_scan", "image_scan"
```

Comment:
@agrare i dont see is_availabe check for image_scan button in this PR, has_active_proxy? only checks if there is an active proxy on selected Image, if selected Image has an active proxy Smartstate Analysis button will be active & displayed for GCE image record